pkgs/top-level/aliases.nix

```diff
@@ -1294,6 +1294,10 @@ mapAliases ({
   tokyo-night-gtk = tokyonight-gtk-theme; # Added 2024-01-28
   tootle = throw "'tootle' has been removed as it is not maintained upstream. Consider using 'tuba' instead"; # Added 2024-02-11
   tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
+  transmission = lib.warn "transmission has been replaced by transmission_3 and transmission_4, which are incompatible, please choose one and backup your data if you upgrade from 3 to 4" transmission_3; # Added 2024-06-10
+  transmission-gtk = lib.warn "transmission-gtk has been replaced by transmission_3-gtk and transmission_4-gtk, which are incompatible, please choose one and backup your data if you upgrade from 3 to 4" transmission_3-gtk; # Added 2024-06-10
+  transmission-qt = lib.warn "transmission-qt has been replaced by transmission_3-qt and transmission_4-qt, which are incompatible, please choose one and backup your data if you upgrade from 3 to 4" transmission_3-qt; # Added 2024-06-10
```

Comment:
Maybe also mention the possible data loss and point to release notes?

Reply:
> Maybe also mention the possible data loss and point to release notes?

I was thinking about it but the warning became too long so I wasn't sure ... We don't care if it becomes too long? We do have the release notes after all...

Reply:
With the wording I am proposing above (edited), pointing to release notes makes it just 30% longer. Since we are ignoring the `nix-env` issue, I do not think we are that size constrained – people will just see the two lines of deprecation notice and then fix their config if they are annoyed.

Reply:
Very well phrased. I used that.